app/models/submitter.rb

```diff
@@ -119,6 +119,14 @@ def with_signature_fields?
     end
   end
 
+  def current_documents
+    return documents if document_generation_events.complete.none?
+
+    # Use completed_at as the marker since documents are generated after completion
+    # This handles re-completion: only returns documents from the latest completion
+    documents.where(active_storage_attachments: { created_at: completed_at.. })
```

Comment:
Rubocop preferred `completed_at..` to `completed_at >= ?, completed_at`. So this returns documents created at or after the completed_at time.